### DISCUSSION TITLE
Play pause

### DISCUSSION THREAD

**Discussion by Muscule** [chaos-mesh/chaos-mesh#2166] [Q&A]
where is the button in v2.0.0?

**Comment by STRRL** [comment]
We have updated the spec of PodChaos, and if you create an experiment of one PodChaos, it only acts as a "one-shot".

You should use Schedule with Podchaos, the `cron` is moved into Schedule.

**Reply by Muscule** [reply]
Why? It is not comfortable.

**Reply by fewdan** [reply]
Because killing a pod can be done in an instant, it is impossible to pause chaos during the process of killing a pod.
